Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -33,10 +33,11 @@ public class BoundaryManager : Singleton<BoundaryManager>
         // This puts the origin (0, 0, 0) on the floor if a floor has been established during setup via MixedRealityPortal.
         private TrackingSpaceType opaqueTrackingSpaceType = TrackingSpaceType.RoomScale;
 
-        [SerializeField]
+        //[SerializeField]
+        // Removed for now, until the HoloLens tracking space type story is more clear.
         // Defaulting coordinate system to Stationary for transparent headsets, like HoloLens.
         // This puts the origin (0, 0, 0) at the first place where the user started the application.
-        private TrackingSpaceType transparentTrackingSpaceType = TrackingSpaceType.Stationary;
+        //private TrackingSpaceType transparentTrackingSpaceType = TrackingSpaceType.Stationary;
```

Comment:
The non-package parts of this PR look like they should be added to the dev branch instead of into master.

Reply:
Well I already added these changes to the packages and they're already released, soo....

Reply:
The packages attached to the release aren't actually a snapshot of master...? That seems counter to the entire idea of a release.

Those packages can also be updated to the correct ones.

Reply:
Well the stabiliazation branch got merged into master before I could review it, and this was the stuff that needed to be fixed.  I don't like making releases that include warnings or issues building, which is what the changes outlined show.

Reply:
Packages need to be from master. Is that the case?

Reply:
They are. With the fixes addressed.